config/settings.yml

```diff
@@ -69,6 +69,8 @@ plugins:
   chatbot_request_temperature:
     client: false
     default: 70
+    minimum: 0
```

Comment:
These must follow the convention in core.  Literally `min:` and `max:`

Reply:
You are right. Pushing now.